scout/server/blueprints/genes/controllers.py

```diff
@@ -52,16 +52,16 @@ def gene(store, hgnc_id):
     return res
 
 
-def genes_to_json(store, query):
+def genes_to_json(store, query, build):
     """Fetch matching genes and convert to JSON."""
-    gene_query = store.hgnc_genes(query, search=True)
-    json_terms = [
-        {
+    gene_query = store.hgnc_genes(query, build, search=True)
```

Comment:
👍

Reply:
its basically passing on the build argument. I debugged the code at some time so there might appear to be more different than it actually is